docs/source/releases.md

```diff
@@ -5,15 +5,16 @@ We are aiming to ship a new release approximately every 6 weeks. The following r
 
 | Release | Code freeze | General availability |
 | :-----: | :---------: | :------------------: |
-| 1.7     | 2021-01-05  | 2021-01-19           |
+| 1.8     | 2022-02-24  | 2021-03-10           |
```

Comment:
Would you mind making it a Wed as we usually do? That make code freeze fall on 2.3. and GA on 16.3.